components/DisclosureCard.tsx

```
@@ -101,7 +132,79 @@ export function DisclosureCard({ title, content, image }: DisclosureCardProps) {
+                        <DrawerClose asChild>
+                          <div className="flex justify-between items-center">
+                            <div>
+                              <Button className="w-24">Close</Button>
+                            </div>
+                          </div>
+                        </DrawerClose>
+                      </DrawerFooter>
+                    </DrawerContent>
+                  </Drawer>
+                )}
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)